apps/test/unit/code-studio/pd/permissionTest.js

```diff
@@ -1,91 +1,81 @@
 import {expect} from 'chai';
-import Permission from '@cdo/apps/code-studio/pd/permission';
+import Permission, {
+  WorkshopAdmin,
+  Facilitator,
+  Organizer,
+  ProgramManager,
+  Partner
+} from '@cdo/apps/code-studio/pd/workshop_dashboard/permission';
 
 describe("Permission class", () => {
-  const setGlobalPermissionString = (permissionString) => {
-    window.dashboard = {
-      workshop: {
-        permission: permissionString
-      }
-    };
-  };
-
   it("Detects workshop admin", () => {
```

Comment:
I feel like all of this is DRY-able.

```
permission_types = ['WorkshopAdmin', 'Facilitator', 'Organizer', 'ProgramManager', 'Partner'];

permission_types.each ((type) => {
const permission = new Permission([type])
expect(permission.has(type)).to.be.true;
permission_types - type.each((other_type) => {
expect(permission.hash(other_type)).to.be.false;
})
```

Something along those lines

Reply:
Agreed, but I didn't change this test, just got it to work with the new format. I'd like to clean it up in a future PR since this one is already large.